doc/changelog/07-vernac-commands-and-options/14208-warn-unqualified-typeclass-instance.rst

```diff
@@ -0,0 +1,7 @@
+- **Deprecated:**
+  Similarly to hints, typeclass instances added outside of sections
+  without an explicit locality now generate a deprecation warning. See
+  the hint documentation for more background
```

Comment:
```suggestion
  Like hints, typeclass instances added outside of sections
  without an explicit locality now generate a deprecation warning. See
  :ref:`XXXXXXX`
```

The hyperlink should go to type-classes.rst, presumably to the text you add there.  It shouldn't mention "hint documentation"

Reply:
Done.